### PR TITLE
Add missing RID operator descriptions

### PR DESCRIPTION
**Pull request by Mickeon**
Also tweaks the descriptions above a bit.

### CODE REVIEW

File: doc/classes/RID.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		Handle for a [Resource]'s unique ID.
 	</brief_description>
 	<description>
-		The RID type is used to access the unique integer ID of a resource. They are opaque, which means they do not grant access to the associated resource by themselves. They are used by and with the low-level Server classes such as [RenderingServer].
+		The RID type is used to access the unique integer ID of a [Resource]. They are opaque, which means they do not grant access to the associated resource by themselves. They are used by and with the low-level server classes, such as [DisplayServer] and [RenderingServer].
```
Comment (bruvzg) [inline doc/classes/RID.xml:7]: > integer ID of a [Resource]

It's not always true, server can have RIDs of internal objects which are not `Resource`s.

Comment (Mickeon) [reply doc/classes/RID.xml:7]: Well, ain't that some misleading information sitting there for plenty of time...

Comment (Mickeon) [reply doc/classes/RID.xml]: Took this occasion to change the above descriptions a bit more. See what you think.

File: doc/classes/RID.xml [hunk 2]
```diff
@@ -42,36 +43,42 @@
...
 			<param index="0" name="right" type="RID" />
 			<description>
+				Returns [code]true[/code] if the [RID]'s ID is greater than [param right]'s ID.
 			</description>
 		</operator>
 		<operator name="operator &gt;=">
 			<return type="bool" />
 			<param index="0" name="right" type="RID" />
 			<description>
+				Returns [code]true[/code] if the [RID]'s ID is greater or equal than [param right]'s ID.
```
Comment (mhilbrunner) [inline doc/classes/RID.xml:81]:
```suggestion
				Returns [code]true[/code] if the [RID]'s ID is greater than or equal to [param right]'s ID.
```

File: doc/classes/RID.xml [hunk 3]
```diff
@@ -42,36 +43,42 @@
...
 			<param index="0" name="right" type="RID" />
 			<description>
+				Returns [code]true[/code] if the [RID]'s ID is less than [param right]'s ID.
 			</description>
 		</operator>
 		<operator name="operator &lt;=">
 			<return type="bool" />
 			<param index="0" name="right" type="RID" />
 			<description>
+				Returns [code]true[/code] if the [RID]'s ID is less or equal than [param right]'s ID.
```
Comment (mhilbrunner) [inline doc/classes/RID.xml:60]:
```suggestion
				Returns [code]true[/code] if the [RID]'s ID is less than or equal to [param right]'s ID.
```

**Review by YuriSizov — APPROVED** [review]
LGTM. Most of it is pretty generic, though such are comparison operators. The change to the description drawing a distinction between `Resource` as a high-level class and a low-level resource-like entity, object seems fine to me.

**Comment by Mickeon** [comment]
I myself do not exactly understand the point of the "less than" and "greater than" operators, unless they are somewhat telling of how "old" a low-level object is.

**Comment by akien-mga** [comment]
> I myself do not exactly understand the point of the "less than" and "greater than" operators, unless they are somewhat telling of how "old" a low-level object is.

There's no practical use for end users. They're required in all Variants for the implementation of `sort()`. E.g. if you call `sort()` on an Array of RIDs, it needs a deterministic way to handle them.

**Pull request merged**